Earthfile

```diff
@@ -284,12 +283,15 @@ earthly-docker:
 earthly-integration-test-base:
     FROM +earthly-docker
     ENV EARTHLY_CONVERSION_PARALLELISM=5
-    RUN earthly config global.disable_analytics true
-    RUN earthly config global.local_registry_host 'tcp://127.0.0.1:8371'
+    ENV GLOBAL_CONFIG={disable_analytics: true, local_registry_host: 'tcp://127.0.0.1:8371'}
+    ENV NO_DOCKER=1
+    ENV SRC_DIR=/test
```

Comment:
The above options are ones from the new image that would also be user-facing.

`GLOBAL_CONFIG` should be self explanatory.
`NO_DOCKER` exists in case you are in an environment where BYOD is not possible, and you want to still build artifacts. It skips the check for connectivity to a live Docker daemon.
`SRC_DIR` lets you override where you will execute earthly from. By default it is `/src`, but here all our tests are in `/test`, which exercises the option.